components/charity/CharityModal.tsx

```diff
@@ -1,15 +1,23 @@
 import React from "react";
 import { Modal, ModalBody, ModalFooter, ModalHeader, Button } from "reactstrap";
 
-import CharityBody from "./modal/CharityBody.jsx";
-import CaseCompBody from "./modal/custom/CaseCompBody.jsx";
-import CaseCompBody1 from "./modal/custom/CaseCompBody2022.jsx";
+import CharityBody from "./modal/CharityBody";
+import CaseCompBody from "./modal/custom/CaseCompBody";
+import CaseCompBody1 from "./modal/custom/CaseCompBody2022";
+import { CharityEvent } from "data/types";
```

Comment:
IDE was complaining at me about "Cannot find module 'data/types', fixed with
```suggestion
import { CharityEvent } from "../../data/types";
```

Same goes for anywhere else you have `import x from "data/types";`

Reply:
Weird - my IDE works just fine, but I will make everything strictly relative.

Reply:
Does all the "components/..." and "style/..." also error for you?

Reply:
<img width="1103" alt="image" src="https://github.com/coopsoc/website/assets/40455550/e863597f-a7a9-4b60-b753-b84ee881fcf9">
yep, not fully sure why

Reply:
fix - upgrade node from v18 to v20
<img width="884" alt="image" src="https://github.com/coopsoc/website/assets/40455550/595c2646-8c2e-4b77-b820-fd7d00a7fc85">